message/testing.go

```diff
@@ -0,0 +1,114 @@
+package message
```

Comment:
This appears to be a move from core/testing.go to message/testing.go but it's showing up as a new file. Did you use `git mv`? It would be nice if we could keep the history.